### DISCUSSION TITLE
Einsatz zum säen ohne Dünger

### DISCUSSION THREAD

**Discussion by c0nnex** [Mogli12/AIVehicleExtension#289] [Ideas]
Wenn ich einen AiVe Helfer zum säen auf ein Feld mit zb. der Amazone Citan schicke, dann weigert er sich beharrlich das zu tun, wenn der Düngertank leer ist, auch wenn das in diesem Fall so gewollt ist. (Nur saat, keine Düngung)
Eine Option "Ignore secondary tank" oder so wäre nice.

**Comment by Mogli12** [comment]
Probier mal die Version 15 aus. Da kann man in den Settings einstellen, dass die Unterfussdüngung unterdrückt wird.

**Comment by c0nnex** [comment]
Mit .15 kann man mit der Amazone bei gesenktem+eingeschaltetem Sähwerk nicht mehr vorwärts fahren, weder mit aive noch von hand. Egal ob Düngertabnk leer oder voll
```
dataS/scripts/vehicles/specializations/FertilizingSowingMachine.lua(59) : attempt to index local 'self' (a function value)
2022-01-15 20:53 Error: Running LUA method 'update'.
```
(kommt sowohl im Server als auch Client log). Sonst keine Fehler im Log

**Reply by c0nnex** [reply]
Korrktur. Fehler im CLient log:
```
2022-01-15 20:58 Error in FS19_AIVehicleExtension.AutoSteeringEngine.checkTools2
2022-01-15 20:58 C:/Users/uli/Documents/My Games/FarmingSimulator2022/mods/FS22_AIVehicleExtension/AutoSteeringEngine.lua:7953: attempt to index field 'upperRotLimit' (a nil value)

```

**Comment by Mogli12** [comment]
Den Fehler habe ich behoben. Aber irgendwie funktioniert das Umschalten nicht so wie erwartet. Aber säen ohne Dünger im Tank hat bei mir geklappt.

**Reply by c0nnex** [reply]
Funktioniert, danke für die schnelle Umsetzung!

habe einige probs mit dem Case Drescher und AiVe, mache ich aber in nem speraten Issue
